ui/component/channelThumbnail/view.jsx

```diff
@@ -58,13 +59,21 @@ function ChannelThumbnail(props: Props) {
     showMemberBadge,
     isChannel,
     checkMembership = true,
+    minOptimization,
   } = props;
   const [thumbLoadError, setThumbLoadError] = React.useState(ThumbUploadError);
   const shouldResolve = !isResolving && claim === undefined;
   const thumbnail = rawThumbnail && rawThumbnail.trim().replace(/^http:\/\//i, 'https://');
   const thumbnailPreview = rawThumbnailPreview && rawThumbnailPreview.trim().replace(/^http:\/\//i, 'https://');
   const defaultAvatar = AVATAR_DEFAULT || Gerbil;
-  const channelThumbnail = thumbnailPreview || thumbnail || defaultAvatar;
+  const thumb = thumbnailPreview || thumbnail || defaultAvatar;
+  // checking with CDN to see if a max size can be passed, but for now, this is an improvement vs not using the optimizer
+  const channelThumbnail = getThumbnailCdnUrl({
+    thumbnail: thumb,
+    width: minOptimization ? 500 : 50,
+    height: minOptimization ? 500 : 50,
+    quality: minOptimization ? 95 : 85,
+  });
```

Comment:
Can ship first

----

**Details:**  If the thumbnail is already hardcoded, then we can remove `OptimizedImage` down below as it becomes pointless.  I think it might end up with 2 optimizer calls if `OptimizedImage` works as intended.
_But there are some fallback/error code embedded within, so maybe not a simple replacement with `<img>` (not sure, need to test)._

Based on the snippet in Slack, I think the issue is with `OptimizedImage` failing to get mounted size in some parent combinations.  Probably just need to find a more robust method.

Reply:
Yeah, can you do that for me? No point in calling it again. Agree with the other approach in the future.